scripts/builtin/img_transform_linearized.dml

```diff
@@ -0,0 +1,88 @@
+#-------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#-------------------------------------------------------------
+
+# The Linearized Image Transform function applies an affine transformation to linearized images.
+# Optionally resizes the image (without scaling).
+# Uses nearest neighbor sampling.
+#
+# INPUT:
+# -------------------------------------------------------------------------------------------
+# img_in       Linearized input images as 2D matrix with top left corner at [1, 1]
+# out_w        Width of the output matrix
+# out_h        Height of the output matrix
+# a,b,c,d,e,f  The first two rows of the affine matrix in row-major order
```

Comment:
each variable have to have a new line in the  documentation

Or maybe these should be combined into the translation matrix by the caller?

Reply:
This was the same interface and the documentation format from the existing implementation.
  
Should I write the descriptions like so?:
a = The element located at [1,1] in the affine matrix

Reply:
well, then the existing is wrong as well.

We have parsers that make the python interface code. it does support this syntax (surprised me now), but does not follow the rules of mapping variables to parameters, and therefore must be breaking something else.

![image](https://github.com/apache/systemds/assets/9947148/e8887d29-c531-46c0-904f-d39586147ea2)

https://apache.github.io/systemds/api/python/api/operator/algorithms.html#systemds.operator.algorithm.img_crop